client/my-sites/importer/importing-pane.jsx

```diff
@@ -14,6 +15,61 @@ import ProgressBar from 'components/progress-bar';
 import MappingPane from './author-mapping-pane';
 import Spinner from 'components/spinner';
 
+const sum = ( a, b ) => a + b;
+
+/*
+ * The progress object comes from the API and can
+ * contain different object counts.
+ *
+ * The attachments will lead the progress because
+ * they take the longest in almost all circumstances.
+ *
+ * progressObect ~= {
+ *     post: { completed: 3, total: 12 },
+ *     comment: { completed: 0, total: 3 },
+ *     …
+ * }
+ */
+const calculateProgress = progress => {
```

Comment:
@rralian these will probably end up as selectors once I have moved this from a Flux-based component into a Redux one. I originally programmed them in the component, but since they don't have any data connections to the component I pulled them out in hopes of doing what you are talking about here.

If you are comfortable with it for now, I would like to leave them in this file, then move them with a Redux overhaul. They could be pulled into another file, but this is the only component that will be using it.

Let me know - either way is fine with me.